WcaOnRails/db/seeds/countries.seeds.rb

```diff
@@ -1,208 +1,5 @@
-('Philippines', 'Philippines', '_Asia', 'PH'),
-('Pitcairn Islands', 'Pitcairn Islands', '_Oceania', 'PN'),
-('Poland', 'Poland', '_Europe', 'PL'),
-('Portugal', 'Portugal', '_Europe', 'PT'),
-('Puerto Rico', 'Puerto Rico', '_North America', 'PR'),
-('Qatar', 'Qatar', '_Asia', 'QA'),
-('Romania', 'Romania', '_Europe', 'RO'),
-('Russia', 'Russia', '_Europe', 'RU'),
-('Saint Kitts and Nevis', 'Saint Kitts and Nevis', '_North America', 'KN'),
-('Saint Lucia', 'Saint Lucia', '_North America', 'LC'),
-('Saint Vincent and the Grenadines', 'Saint Vincent and the Grenadines', '_North America', 'VC'),
-('Samoa', 'Samoa', '_Oceania', 'WS'),
-('San Marino', 'San Marino', '_Europe', 'SM'),
-('Sao Tome and Principe', 'Sao Tome and Principe', '_Africa', 'ST'),
-('Saudi Arabia', 'Saudi Arabia', '_Asia', 'SA'),
-('Senegal', 'Senegal', '_Africa', 'SN'),
-('Serbia', 'Serbia', '_Europe', 'RS'),
-('Sierra Leone', 'Sierra Leone', '_Africa', 'SL'),
-('Singapore', 'Singapore', '_Asia', 'SG'),
-('Slovakia', 'Slovakia', '_Europe', 'SK'),
-('Slovenia', 'Slovenia', '_Europe', 'SI'),
-('Solomon Islands', 'Solomon Islands', '_Oceania', 'SB'),
-('Somalia', 'Somalia', '_Africa', 'SO'),
-('South Africa', 'South Africa', '_Africa', 'ZA'),
-('South Sudan', 'South Sudan', '_Africa', 'SS'),
-('Spain', 'Spain', '_Europe', 'ES'),
-('Sri Lanka', 'Sri Lanka', '_Asia', 'LK'),
-('Sudan', 'Sudan', '_Africa', 'SD'),
-('Suriname', 'Suriname', '_South America', 'SR'),
-('Swaziland', 'Swaziland', '_Africa', 'SZ'),
-('Sweden', 'Sweden', '_Europe', 'SE'),
-('Switzerland', 'Switzerland', '_Europe', 'CH'),
-('Syria', 'Syria', '_Asia', 'SY'),
-('Taiwan', 'Taiwan', '_Asia', 'TW'),
-('Tanzania', 'Tanzania', '_Africa', 'TZ'),
-('Thailand', 'Thailand', '_Asia', 'TH'),
-('Togo', 'Togo', '_Africa', 'TG'),
-('Tonga', 'Tonga', '_Oceania', 'TO'),
-('Trinidad and Tobago', 'Trinidad and Tobago', '_North America', 'TT'),
-('Tunisia', 'Tunisia', '_Africa', 'TN'),
-('Turkey', 'Turkey', '_Europe', 'TR'),
-('Turks and Caicos Islands', 'Turks and Caicos Islands', '_North America', 'TC'),
-('Tuvalu', 'Tuvalu', '_Oceania', 'TV'),
-('Uganda', 'Uganda', '_Africa', 'UG'),
-('Ukraine', 'Ukraine', '_Europe', 'UA'),
-('United Arab Emirates', 'United Arab Emirates', '_Asia', 'AE'),
-('United Kingdom', 'United Kingdom', '_Europe', 'GB'),
-('Uruguay', 'Uruguay', '_South America', 'UY'),
-('USA', 'USA', '_North America', 'US'),
-('Uzbekistan', 'Uzbekistan', '_Asia', 'UZ'),
-('Vanuatu', 'Vanuatu', '_Oceania', 'VU'),
-('Venezuela', 'Venezuela', '_South America', 'VE'),
-('Vietnam', 'Vietnam', '_Asia', 'VN'),
-('XA', 'Multiple Countries (Asia)', '_Asia', 'XA'),
-('XE', 'Multiple Countries (Europe)', '_Europe', 'XE'),
-('XS', 'Multiple Countries (South America)', '_South America', 'XS'),
-('Yemen', 'Yemen', '_Asia', 'YE'),
-('Zambia', 'Zambia', '_Africa', 'ZM'),
-('Zimbabwe', 'Zimbabwe', '_Africa', 'ZW');"
-ActiveRecord::Base.connection.execute(sql)
+Country::ALL_STATES.each do |state|
+  Country.create!(state)
```

Comment:
Actually, what about changing `ALL_SORTED_BY_LOCALE` to populate itself from the json config file instead of from the database?